usaspending_api/etl/es_transaction_template.json

```diff
@@ -1,10 +1,10 @@
 {
-  "index_patterns": ["*"],
+  "index_patterns": ["transactions*"],
```

Comment:
Is there a reason this is changed here? `es_configure` already modified this value based upon `settings.TRANSACTIONS_INDEX_ROOT`

Reply:
Yes, as we discussed Friday. Naming conventions will be consistent and settings.TRANSACTIONS_INDEX_ROOT for ingest and querying will NOT start with transactions.

Reply:
If this is a different prefix, then:

1. Please set in settings.py instead of buried in django management commands
2. Alter `es_configure.py` so that is uses the new alias properly